Zend/tests/bug39602.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 Bug #39602 (Invalid session.save_handler crashes PHP)
---SKIPIF--
-<?php if (!extension_loaded("session")) die("skip session extension required"); ?>
```

Comment:
I don't think the session extension is required.

Reply:
This is the purpose of this patch: remove skipif sections that were blocking this tests from being running when these extensions weren't available.

Reply:
I see what you mean. However, as this test is testing a session ini option it does not make sense to run it without the session extension. It will pass, but it will also be useless, because it does not test anything.

Reply:
So we should keep the `SKIPIF` sections, or drop the test?

Reply:
Just keep the `SKIPIF` section. :)